dusty/commands/setup.py

```diff
@@ -64,6 +65,20 @@ def _get_and_configure_nginx_includes_dir():
     _get_raw_input('\n'.join(textwrap.wrap('You have a custom nginx config setup. Could not find an nginx.conf file. Please read our docs to see what is needed for the nginx config.  Once you have figured it out, please use `dusty config` command to adjust your `nginx_includes_dir`', 80)))
     return ''
 
+def _get_boot2docker_vm_size():
+    # all math is done in megabytes
+    memory_megs = virtual_memory().total / 2**20
+    if memory_megs >= 16 * 2**10:
+        vm_megs = 6 * 2**10
+    elif memory_megs >= 8 * 2**10:
+        vm_megs = 4 * 2**10
+    else:
+        vm_megs = 2 * 2**10
```

Comment:
Generally when you're doing math like this you want to create unit constants to make it easier to read and harder to make mistakes, e.g.

```
GIGABYTES = 2**10

vm_megs = 6 * GIGABYTES
```